jablib/src/main/java/org/jabref/logic/importer/Importer.java

```diff
@@ -49,6 +49,10 @@ public abstract class Importer implements Comparable<Importer> {
      * @throws IOException Signals that an I/O exception has occurred.
      */
     public boolean isRecognizedFormat(Path filePath) throws IOException {
+        if (!Files.exists(filePath) || !Files.isRegularFile(filePath)) {
```

Comment:
The latter includes the former, doesn't it?

```suggestion
        if (!Files.isRegularFile(filePath)) {
```